sakura/user.css

```
@@ -0,0 +1,226 @@
+
+/* Main Background */
+.Root__main-view {
+    background:
+        linear-gradient(
+            rgba(10,8,10,.92),
+            rgba(10,8,10,.95)
+        ),
+        url("images/sakura-bg.jpg")
+        !important;
+
+    background-size: cover !important;
```

Stylelint (17.12.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)